src/main/java/org/mvel2/asm/Opcodes.java

```diff
@@ -58,6 +58,7 @@
     int V1_6 = 0 << 16 | 50;
     int V1_7 = 0 << 16 | 51;
     int V1_8 = 0 << 16 | 52;
+    int V1_9 = 0 << 16 | 53;
```

Comment:
I agree that for now this is the best (and quickest) thing we could do to start supporting Java 9, but I'm also afraid that this won't be enough. I'll merge it for now.